### PR TITLE
Minor improvements and addition of blog index

### PR DESCRIPTION
**Pull request by phae**
Hello!

### Why did I do this?

I'm hoping to start writing some more content for the blog, so I needed to get that in a slightly better shape. The goal is to get more people talking about and linking to the OWA work (and generally seeing how busy and productive you all are!). 

### I've done a few things:

- Added an index of blog posts to the /blog page, so there's something actually there
- Pulled the promo block into it's own partial so it can be used alone
- Fiddled with some of the titles so they're more informative (mostly @brucelawson's - hope you don't mind)

Also, there's a couple of new helpers that Alex R added:
`npm run build` which is an alias for npm run production
`npm run clean` which will delete the built output

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -685,7 +685,7 @@ header nav ul li:last-of-type a:hover {
 }
 
 
-/* ------------------------  HOME ---------------------------- */
+/* ------------------------  HOME -------------`--------------- */
```
Comment (sbesh91) [inline src/css/main.css:688]: Is the extra tick ` on purpose here?

**Review by sbesh91 — APPROVED** [review]

File: src/css/main.css (hunk 1 above)
Comment (phae) [reply src/css/main.css:688]: hah, no. must have been a fat-finger

**Comment by phae** [comment]
Thank you, @sbesh91! Happy to have this merged at someone's earliest convenience.

**Comment by sbesh91** [comment]
> Thank you, @sbesh91! Happy to have this merged at someone's earliest convenience. 

I have no qualms.  Can you merge with just my approval?

**Comment by phae** [comment]
@sbesh91 No, but I'd be happy to if you can grant me permission to do so on this repo.

**Pull request merged**